src/metatensor/models/experimental/pet/tests/test_exported.py

```diff
@@ -16,14 +16,13 @@
 
 
 @pytest.mark.parametrize("device", ["cpu", "cuda"])
-@pytest.mark.parametrize("dtype", [torch.float32, torch.float64])
-def test_to(tmp_path, device, dtype):
+def test_to(tmp_path, device):
     """Tests that the `.to()` method of the exported model works."""
     if device == "cuda" and not torch.cuda.is_available():
         pytest.skip("CUDA is not available")
 
     os.chdir(tmp_path)
-
+    dtype = torch.float32  # for now
```

Comment:
Does it not work anymore with float64?

Reply:
Well, after having a discussion with Sergey, I decided to disable this test for now. Most likely, PET will only be working with float32 and maybe with bfloat16 in the future

Reply:
Ok, but it doesn't cost us much to make it work with float64 once exported

Reply:
Yeah, you are right
In fact, the main issue is that in the current implementation of PET, there are some declaration of tensors that use a specific dtype (like torch.FloatTensor)

I’ve recently made a branch in PET and cleaned up this a bit, so as long as it is merged - we can return the float64 support back

Reply:
Ok, perfect. Everything is fine by me. IMO training in float32 is completely fine (since the results seem to be very good), but I would keep the option to move it to float64 as a post-processing step for MD